packages/ui/src/components/elements/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import cx from "clsx";
+
+import ButtonProps from "./Button.props";
+import { UI_CLASSNAME } from "../../..";
+
+function Button({
+  className,
+  variant,
+  inverted,
+  disabled,
+  label,
+}: ButtonProps) {
+  //TODO: Add your code here
+  return (
+    <div>
```

Comment:
@frankjlin16 is there a use case for this outer `div`? If not can it be removed?